audio/CMakeLists.txt

```diff
@@ -114,52 +127,86 @@ if(AUDIODRV_PULSE)
 	set(AUDIO_INCLUDES ${AUDIO_INCLUDES} ${PULSEAUDIO_INCLUDE_DIR})
 	# There seems to be no package or variable for pulse-simple.
 	set(AUDIO_LIBS ${AUDIO_LIBS} pulse-simple ${PULSEAUDIO_LIBRARY})
+	set(VGMAUDIO_PKGCONFIG_LDFLAGS ${VGMAUDIO_PKGCONFIG_LDFLAGS} "-lpulse-simple")
 endif()
 
 if(AUDIODRV_APPLE)
 	set(AUDIO_DEFS ${AUDIO_DEFS} " AUDDRV_CA")
 	set(AUDIO_FILES ${AUDIO_FILES} AudDrv_CoreAudio.c)
 	set(AUDIO_LIBS ${AUDIO_LIBS} "-framework AudioToolbox")
+	set(VGMAUDIO_PKGCONFIG_LDFLAGS ${VGMAUDIO_PKGCONFIG_LDFLAGS} "-framework AudioToolbox")
 endif()
 
 if(AUDIODRV_LIBAO)
 	find_package(LibAO REQUIRED)
 	set(AUDIO_DEFS ${AUDIO_DEFS} " AUDDRV_LIBAO")
 	set(AUDIO_FILES ${AUDIO_FILES} AudDrv_libao.c)
 	set(AUDIO_LIBS ${AUDIO_LIBS} libao::libao)
+	set(VGMAUDIO_PKGCONFIG_LDFLAGS ${VGMAUDIO_PKGCONFIG_LDFLAGS} "-lao")
```

Comment:
I would prefer getting the actual library name from the *libao::libao* target instead of hardcoding it for pkg-config.
But if that should be be a huge hassle, I'm also fine leaving it as is.

Reply:
Unfortunately I'm not sure how to get that. The only property I can find is `IMPORTED_LOCATION`, but that's the actual path to the library. If I'm building a linux package, that path isn't useful for a pkg-config file.

It may be better to add `ao` to the pkg-config file `Requires.private` property instead, since libao is very likely to have a `pkg-config` file installed.

Reply:
Changed to use the `pkg-config` names in the `Requires.private` section, at least for the libraries that should have `pkg-config` files.